docs/modules/node.md

```diff
@@ -607,6 +607,76 @@ node.startupcommand("=if LFS.init then LFS.init() else dofile('init.lua') end")
 ```
 
 
+## node.startupcounts()
+
+Query the performance of system startup.
+Note that this is only available if
```

Comment:
Please wrap this with an admonition like we e.g. do for `wifi.startsmart()` https://nodemcu.readthedocs.io/en/latest/modules/wifi/#wifistartsmart. I doubt we are consistent as to the admonition _type_ (note, important, etc.).

Reply:
Fixed.